random/random.mbti

```diff
@@ -3,13 +3,19 @@ package moonbitlang/core/random
 // Values
 
 // Types and methods
-type RandomState
-impl RandomState {
-  gen_double(Self) -> Double
-  gen_int(Self) -> Int
-  gen_int64(Self) -> Int64
-  init_state(~seed : Int = ..) -> Self
-  new() -> Self
+type Rand
+impl Rand {
+  double64(Self) -> Double
```

Comment:
just double?

Reply:
I tried to add float but float show is not implemented and thus cannot be tested, so I git stashed it and forgot..